### PR TITLE
Fix Safari issue, add unit tests

### PR DESCRIPTION
**Pull request by DimaNaumov**

### CODE REVIEW

File: src/globemap.ts [hunk 1]
```diff
@@ -1511,5 +1516,14 @@ module powerbi.extensibility.visual {
 
             return controlsContainerSVG;
         }
+
+        public static getCategoricalValueByIndex(category: DataViewCategoryColumn, index: number): string {
```
Comment (ignatvilesov) [inline src/globemap.ts]: This method doesn't return string if value is actually number.
Please use ES2015 template string to convert to string.

File: src/globemap.ts [hunk 2]
```diff
@@ -430,8 +431,12 @@ module powerbi.extensibility.visual {
                             : undefined;
                         toolTipDataLongName = categorical.X && categorical.X.source && categorical.X.source.displayName;
                         toolTipDataLatName = categorical.Y && categorical.Y.source && categorical.Y.source.displayName;
+                        locationValue = "";
                     }
 
+                    const longitudeValue = GlobeMap.getCategoricalValueByIndex(categorical.X, i);
```
Comment (ignatvilesov) [inline src/globemap.ts]: Please specify a type

File: src/globemap.ts [hunk 3]
```diff
@@ -430,8 +431,12 @@ module powerbi.extensibility.visual {
...
+                    const latitudeValue  = GlobeMap.getCategoricalValueByIndex(categorical.Y, i);
```
Comment (ignatvilesov) [inline src/globemap.ts]: Please specify a type

File: src/globemap.ts [hunk 4]
```diff
@@ -414,14 +414,15 @@ module powerbi.extensibility.visual {
                     let toolTipDataLongName: string;
                     let toolTipDataLatName: string;
                     let location: ILocation;
-
+                    let locationValue: string;
```
Comment (ignatvilesov) [inline src/globemap.ts]: This is unused variable. Please remove

File: src/globemap.ts (hunk 1 above)
Comment (DimaNaumov) [reply src/globemap.ts]: Fixed

File: src/globemap.ts (hunk 4 above)
Comment (DimaNaumov) [reply src/globemap.ts]: Fixed

**Pull request merged**